transports/Dockerfile

```
@@ -1,64 +1,50 @@
-# --- Second Stage: Runtime image ---
-FROM alpine:latest
-WORKDIR /app
-
-# Copy the compiled binary from the builder stage
-COPY --from=builder /app/main .
-# Ensure the binary is executable
-RUN chmod +x /app/main
-# Create a directory to store configuration files
-RUN mkdir -p /app/config
+# Move binary to app directory
+RUN mv $(go env GOPATH)/bin/linux_amd64/bifrost-${TRANSPORT_TYPE} /app/main
```

Hadolint (2.12.0)

[warning] Quote this to prevent word splitting.

(SC2046)